docs/tutorials/get-started/single-component-tutorials-servo-mousemover.md

```diff
@@ -0,0 +1,274 @@
+
+![A screenshot of the word Language with four boxes below it with Python, Golang, Typescript (Web), and Remotes in each one. Python has a black background with white text and a checkmark, the other three have white backgrounds with black text.](../../img/single-component-tutorials-servo-mousemover/choose-python.png)
+
+Since we already installed the Python SDK, we can skip the first step. Copy the code in from step 2.
+
+![A screenshot of the Viam app showing the CODE SAMPLE tab. On this tab it has Language: with four boxes, Python is black background with white text and has a checkmark indicating it is selected, then there's Golang, Typescript (Web), and Remotes with white backgrounds and black text. The page next says Python SDK (Documentation) with a link to Documentation. Below that there\'s a place that says "pip install viam-SDK" this is how you install the Viam Python SDK. Below that instruction is the instructions to "Then copy and run the following code to connect to your robot:" with 7 lines of code seen (the screenshot is incomplete).](../../img/single-component-tutorials-servo-mousemover/code-sample-page.png)
+
+The copied the code needs to go in a Python file on the Raspberry Pi.
+You can do so by creating a file directly inside the Raspberry Pi and editing the file with nano.
+
+Inside the Raspberry Pi Terminal, run the following command to create a folder to put our files in (name this folder whatever you want).
+
+`mkdir mousefolder`
+
+Go into mousefolder.
+
+`cd mousefolder`
+
+Create a file using nano with the .py which is the python file extension, (name this file whatever you want).
+
+`nano anyname.py`
+
+Paste the code you got from the **CODE SAMPLE** tab in the Viam app. Press CTRL+O, then CTRL+M, then CTRL+X to save the code and exit.
+
+![Raspberry pi computer terminal with black background and mostly white, but there's some green, teal, purple and red parts of the text. At the top it says GNU nano 5.4 anyname.py. Inside the terminal it shows the connect code from the previous example, but this time it's in the terminal. At the bottom it shows the settings for this nano program.](../../img/single-component-tutorials-servo-mousemover/sample-code.png)
+
+Now, run the code using the below command to get the resource information that lets us know if the connection is good or if there are any errors.
+
+`python3 anyname.py`
+
+![Raspberry pi computer terminal with black background and mostly white, but there's one teal word: "INFO" located at the top next to viam.rpc.dial (dial.py:209). The code shown below that is white text and shows Viam's RDK Resources, and it shows no error.](../../img/single-component-tutorials-servo-mousemover/resources-printed.png)
+
+There are no errors in the resources above, so we have confirmed a good software connection between the Python SDK and your Viam robot!
```

Comment:
```suggestion
If your program outputs no errors, your code can connect to your robot correctly.
```

This prints out resources that the user hasn't configured yet. Also I cannot imagine what an error in the resource would be here. The suggestion makes that clearer